querydsl-core/src/test/java/com/querydsl/core/group/GroupByMapTest.java

```diff
@@ -45,6 +36,30 @@ public void Group_Order() {
     }
 
     @Test
+    public void Set_By_Sorted() {
+        Map<Integer, Group> results = BASIC_RESULTS
+                .transform(groupBy(postId).as(postName, sortedSet(commentId)));
+
+        Group group = results.get(1);
+        Iterator<Integer> it = group.getSet(commentId).iterator();
+        assertEquals(1, it.next().intValue());
+        assertEquals(2, it.next().intValue());
+        assertEquals(3, it.next().intValue());
+    }
+
+    @Test
+    public void Set_By_Sorted_Reverse() {
+        Map<Integer, Group> results = BASIC_RESULTS
```

Comment:
Could you maybe use `BASIC_RESULTS_UNORDERED` here instead of the already sorted `BASIC_RESULTS`?

The same for `Set_By_Sorted()`.